bin/citrea/tests/e2e/mod.rs

```diff
@@ -1560,7 +1560,8 @@ async fn test_system_tx_effect_on_block_gas_limit() -> Result<(), anyhow::Error>
     // send 41 ether transfer transactions
     let addr = Address::from_str("0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266").unwrap();
 
-    for _ in 0..41 {
+    // @ercecan we need to recalculate here
```

Comment:
Did the gas usage of sys txs change again?

Reply:
no there is an extra sys tx now